test/e2e/volume_provisioning.go

```diff
@@ -107,7 +107,7 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 
 	framework.KubeDescribe("DynamicProvisioner", func() {
 		It("should create and delete persistent volumes [Slow] [Volume]", func() {
-			framework.SkipUnlessProviderIs("openstack", "gce", "aws", "gke")
```

Comment:
@msau42 Lets add vsphere to the supported provider list for this test.
For Internal vSphere CI, we can use local branch copy, until I cross port all our tests to respective active branches.

cc: @tusharnt, @BaluDontu